salt/states/idem.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Tyler Johnson <tjohnson@saltstack.com>
+#
+
+"""
+Idem Support
+============
+
+This state provides access to idem states
+"""
+import logging
+import pathlib
+import sys
+
+import salt.utils.idem as idem
+
```

Comment:
Maybe utils should be called through the `__utils__` dunder instead of direct imports?

@s0undt3ch What is the recommended convention for that?